docs/parsers.md

```diff
@@ -0,0 +1,71 @@
+# Parsers
+
+Starting in version 3.0, you can define custom parsers to parse design token files. This allows you to define your design token files in *any* language you like as long as you can write a parser for it. 
+
+A custom parser matches design token files based on a file path regular expression. It will get the contents of a file as a string and should return an object of the data. 
+
+Custom parsers can be used to keep design token files in YAML, but they can also be used to add extra metadata onto all properties before transforms are run or anything else you can think of modifying on the design tokens before transforms are run.
```

Comment:
I think you are trying to share example use cases here, but this is hard to parse and a bit confusing.